operators/go/build_publish_yaml

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+cd $(dirname $0)
+
+mkdir publish
+
+cp submariner-operator/deploy/namespace.yaml publish/submariner-namespace.yaml
+
+for f in submariner-operator/deploy/crds/*_crd.yaml
+do
+    echo ---
+    cat $f
+done > publish/submariner-crds.yaml
+
+for f in submariner-operator/deploy/service_account.yaml \
+         submariner-operator/deploy/role.yaml \
+         submariner-operator/deploy/role_binding.yaml \
+         submariner-operator/deploy/operator.yaml
+do
+    echo ---
+    cat $f
+done > publish/submariner-operator.yaml
```

Comment:
Is it possible to do it in a single yaml or are there issues with the ordering?

Reply:
We need to split out the namespace creation, since the operator descriptor itself goes in the `submariner` namespace. The CRDs are installed globally, I haven’t tried installing them with a namespace specifier in combination with other files...